src/main/java/in/twizmwaz/cardinal/module/modules/stats/Stats.java

```diff
@@ -0,0 +1,151 @@
+package in.twizmwaz.cardinal.module.modules.stats;
+
+import in.twizmwaz.cardinal.GameHandler;
+import in.twizmwaz.cardinal.event.MatchEndEvent;
+import in.twizmwaz.cardinal.module.Module;
+import in.twizmwaz.cardinal.module.modules.team.TeamModule;
+import in.twizmwaz.cardinal.util.TeamUtils;
+import org.bukkit.Bukkit;
+import org.bukkit.ChatColor;
+import org.bukkit.Material;
+import org.bukkit.entity.Player;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.entity.PlayerDeathEvent;
+
+import java.util.*;
+
+public class Stats implements Module {
+
+
+    private ArrayList<Tracker> stats = new ArrayList<>();
```

Comment:
It isn't a big deal, but generally one would make the field type List<> here, and then instantiate the ArrayList.